ControlFlow/Program.cs

```diff
@@ -48,39 +73,88 @@ public static string LandscapeOrPortrait(int width, int height)
         // Suspended" instead. Don't Worry about input validation.
         public static string SpeedTrap(int speedLimit, int speedCar)
         {
-            return default;
+             int demeritPoints = (speedCar - speedLimit) / 5;
+
+            if (speedCar > speedLimit && demeritPoints <= 12)
+            {
+                return demeritPoints.ToString();
+            }
+            else if (demeritPoints > 12)
+            {
+                return "License Suspended";
+            }
+            else
+            {
+                return "Okay";
+            }
         }
 
+
+
         // Part 2, Control Flow. Come back to this section after completing the For/Foreach/While loops.
 
         // 2.1 - Write a method to count how many integers between two numbers are divisible by a divider with
         // no remainder. Return the count. For example, if minNumber = 1 and maxNumber = 6, and the divider is 3 then
         // output should be 2.
         public static int DivisibleByNumber(int minNumber, int maxNumber, int divider)
         {
-            return default;
+            var result = 0;
+            for (var i = minNumber; i <= maxNumber; i++)
+            {
+                if (i % divider == 0)
+                {
+                    result++; 
+                }
+            }
+
+            return result;
         }
 
         // 2.2 - Write a method to calculate the sum of all the integers and return it. For example if
         // the program enters (2, 3, 5, 1) then the return should be 11.
         public static int SumIntegers(params int[] numbers)
         {
-            return default;
+            var total = 0;
+
+            foreach (var number in numbers)
+            {
+                total += number;
+            }
+
+            return total;
         }
 
         // 2.3 - Write a method to compute the factorial of an integer return it. For example, if the
         // user enters 5, the program should calculate 5 x 4 x 3 x 2 x 1 and return it as 120. Make sure
         // that 0! = 1.
         public static int FindFactorial(int number)
         {
-            return default;
+            var result = 1;
+
+            while (number != 0)
+            {
+                result *= number;
+                number--; 
+            }
+
+            return result;
         }
 
         // 2.4 - Write a method that takes an array of integers. Then find the maximum of the numbers
         // and return it. For example, if the numbers are {5, 3, 8, 1, 4}, the program should return 8.
         public static int MaximumNumber(params int[] numbers)
         {
-            return default;
+            var result = numbers[0];
+
+            for (var i = 0; i < numbers.Length - 1; i++)
```

Comment:
This is a minor change, but I recommend changing the `var i = 0` to `var i = 1` as you already indexed the result variable as 0 on the previous line.